cfgov/v1/__init__.py

```diff
@@ -15,9 +16,25 @@ def environment(**options):
     env.globals.update({
         'static': staticfiles_storage.url,
         'reverse': reverse,
+        'render_stream_child': render_stream_child
     })
     env.filters.update({
         'slugify': slugify,
         'richtext': wagtailcore_tags.richtext
     })
     return env
+
+
+@contextfunction
+def render_stream_child(context, stream_child):
```

Comment:
oh lol it didn't, i was referring to when i had tried to implement jinja2 integration awhile back. Thought they would have cleaner solution in 1.2 but wasn't the case

https://github.com/cfpb/cfgov-refresh/commit/e45022da202b26834199841c37c1d155a57ed14b